tests/data/flamegraph/colors/java-perl.svg

```diff
@@ -1,1442 +0,0 @@
-<?xml version="1.0" standalone="no"?>
```

Comment:
Why has this entire file been removed?

Reply:
There weren't any tests using this or `js-perl.svg`.